### PR TITLE
feat: Backlog/clean up style

### PR DESCRIPTION
**Pull request by lluisFtrack**
Resolves : 

* CLICKUP-
* FT-a1bd0c1a-1a8c-40e5-8a3a-d2c1b5c55cf7
* SENTRY-
* ZENDESK-

- [ ] I have added automatic tests where applicable.
- [ ] The PR contains a description of what has been changed.
- [ ] The description contains manual test instructions.
- [ ] The PR contains update to the release notes.
- [ ] The PR contains update to the documentation.

This PR has been tested on :

- [ ] Windows.
- [ ] MacOs.
- [ ] Linux.

## Changes
- Clean up style files.
- Added a test UI on test/resources to check style. Can be run with the standalone_ui_test.py Changing the WHAT variable to style_test

Screenshot 2024-04-08 at 11 22 16
Screenshot 2024-04-08 at 11 22 33

### CODE REVIEW

**Comment by henriknorin-ftrack** [comment]
Opener does not match the previous look, see screenshot with feedback.
opener

**Comment by henriknorin-ftrack** [comment]
Publisher also have some artefacts:

**Comment by henriknorin-ftrack** [comment]
Also some comments on the context selector:
Screenshot 2024-04-09 at 11 13 52

File: projects/framework-common-extensions/dialogs/standard_publisher_dialog.py [hunk 1]
```diff
@@ -69,14 +72,21 @@ def pre_build_ui(self):
...
-        scroll_area_widget_layout = QtWidgets.QVBoxLayout()
-        scroll_area_widget_layout.setContentsMargins(0, 0, 0, 0)
-        self._scroll_area_widget.setLayout(scroll_area_widget_layout)
+        index = self.layout().indexOf(self.tool_widget)
+        run_index = self.layout().indexOf(self.run_button)
+        if index != -1:  # Ensure the widget is found in the layout
+            # Remove the old widget from layout
+            self.layout().takeAt(index)
+            # Insert the new widget at the same position
+            self.layout().insertWidget(index, self._scroll_area)
```
Comment (henriknorin-ftrack) [inline projects/framework-common-extensions/dialogs/standard_publisher_dialog.py:81]: Might cause trouble with PySide6

**Review by henriknorin-ftrack — CHANGES_REQUESTED** [review]
LGTM overall, added some visual feedback and comments

**Comment by lluisFtrack** [comment]
> Publisher also have some artefacts: publisher

Yellow borders are on propouse

**Comment by lluisFtrack** [comment]
Comments addressed, yellow cross and selected items is on propose.

File: projects/framework-common-extensions/dialogs/standard_publisher_dialog.py (hunk 1 above)
Comment (lluisFtrack) [reply projects/framework-common-extensions/dialogs/standard_publisher_dialog.py:81]: We will take a look when in PySide6 in case

**Review by henriknorin-ftrack — APPROVED** [review]
LGTM, tested

**Pull request merged**